### PR TITLE
[#120] Fixed issue 120

### PR DESCRIPTION
**Pull request by japro**
All problems should be fixed now.

### CODE REVIEW

File: plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java [hunk 1]
```diff
@@ -231,7 +232,7 @@ private String arithmetic(String sym0, String sym1, ArithOp op) throws IOExcepti
                 try {
                     int rr0 = Integer.decode(r0);
                     if (rr0 > 0) {
-                        IP = rr0;
+                        IP = (Integer) in.getOperand();
```
Comment (vbmacher) [inline plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java]: why this change?

File: plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java [hunk 2]
```diff
@@ -214,7 +215,7 @@ private String arithmetic(String sym0, String sym1, ArithOp op) throws IOExcepti
                 try {
                     int rr0 = Integer.decode(r0);
                     if (rr0 == 0) {
-                        IP = rr0;
+                        IP = (Integer) in.getOperand();
```
Comment (vbmacher) [inline plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java]: why this change?

Comment (japro) [inline plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java]: When doing a jump, the IP should be set to new address, NOT to the current value of R0 (accumulator).

File: plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java (hunk 1 above)
Comment (japro) [inline plugins/cpu/ram-cpu/src/main/java/net/sf/emustudio/ram/cpu/impl/EmulatorEngine.java]: When doing a jump, the IP should be set to new address, NOT to the current value of R0 (accumulator).

**Comment by vbmacher** [comment]
Thanks!
